ui/app/ducks/swaps/swaps.js

```diff
@@ -593,20 +592,16 @@ export const signAndSendTransactions = (history, metaMetricsEvent) => {
     const usedQuote = getUsedQuote(state)
     const usedTradeTxParams = usedQuote.trade
 
-    const estimatedGasLimit = new BigNumber(
-      usedQuote?.gasEstimate || decimalToHex(usedQuote?.averageGas || 0),
-      16,
-    )
+    const estimatedGasLimit = new BigNumber(usedQuote?.gasEstimate || `0x0`, 16)
     const estimatedGasLimitWithMultiplier = estimatedGasLimit
-      .times(1.4, 10)
+      .times(usedQuote?.gasMultiplier, 10)
```

Comment:
Since this property is optional, we should probably have a fallback. I think 1.5 would be the best default. 1.5 is the buffer we use on all MetaMask gas estimates.

Reply:
Done!